tests/vppoutputencode.cpp

```diff
@@ -103,10 +107,15 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encVideoParams.intraPeriod = encParam->intraPeriod;
     encVideoParams.ipPeriod = encParam->ipPeriod;
     encVideoParams.rcParams.bitRate = encParam->bitRate;
+    encVideoParams.rcParams.targetPercentage = encParam->targetPercentage;
```

Comment:
okay, thanks @xuguangxin ;
how about that:
`if (encParam->targetPercentage != DEFAULT)
    encVideoParams.rcParams.targetPercentage = encParam->targetPercentage;`